packages/formation-react/src/components/PromoBanner/usage.stories.mdx

```diff
@@ -0,0 +1,78 @@
+import { useState } from 'react';
+import { Canvas, Meta, Story } from '@storybook/addon-docs/blocks';
+
+import PromoBanner, { PROMO_BANNER_TYPES } from './PromoBanner';
+
+<Meta title="Library/Banners/PromoBanner/Usage" />
+
+export const Template = args => {
+  const [showBanner, setShowBanner] = useState(false);
+  const onClose = () => setShowBanner(false);
+  return showBanner ? (
+    <>
+      <PromoBanner {...args} onClose={onClose} />
+      <p>See the open banner on the bottom of the page.</p>
+    </>
+  ) : (
+    <button onClick={() => setShowBanner(true)}>Show Banner</button>
+  );
+};
+
+export const defaultArgs = {
+  type: PROMO_BANNER_TYPES.announcement,
+  text: 'This is an announcement.',
+};
+
+
+# Promo banners
+
+Promo banners are fixed content used for dismissible announcements such as new tools, news, etc.
+
+## Benefit announcements
+
+Pertains to content focused on benefits: a new tool, a new online benefit application, etc.
+
+<Canvas>
+  <Story name="Benefit announcements" args={{ ...defaultArgs }}>
+    {Template.bind({})}
+  </Story>
+</Canvas>
```

Comment:
I don't see a strong reason for these to have Stories created for them, since we just want this usage doc for designers to see.

```suggestion
<Template {...defaultArgs} />
```

Using `<strong>` inside of the `Template` instead of the `<p>` wrapper may help make things more obvious since this suggestion would remove the iframe.